cli/src/build/onboarding/ui/components.tsx

```
@@ -2,7 +2,8 @@ import type { FC } from 'react'
 import { Box, Text, useInput } from 'ink'
 import Spinner from 'ink-spinner'
 // src/build/onboarding/ui/components.tsx
-import React, { useState } from 'react'
+import React, { useEffect, useState } from 'react'
+import type { DiffLine } from '../diff-utils.js'
```

ESLint

[error] Expected "../diff-utils.js" (type-parent) to come before "react" (value-external).

(perfectionist/sort-imports)